modules/score_data_to_model.nf

```diff
@@ -1,23 +0,0 @@
-// score submission results for data to model challenges
-process SCORE_DATA_TO_MODEL {
-    tag "${submission_id}"
-    label "flexible_compute"
```

Comment:
Just want to flag that this label is not present in `SCORE` in case this was unintentional.

Reply:
I removed this because it assigns 16 GB to the compute environment, which I don't think we would need for the scoring process (usually it's going to be a simple python script). But I can add this back if you think `score` should have more memory allocation for data-to-model submissions.

Reply:
TBH I don't remember why I [added](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/11) that much compute in the first place. 

I will say that the memory allocated isn't just about the complexity of the processing being done (python script), but also how much data is being worked through/staged. This is probably fine for now but it's possible that individual challenges may need some more specific compute allocated which we can handle in the config case-by-case if the need arises.